proxy_manager.go

```diff
@@ -222,3 +224,33 @@ func (m *proxyManager) getFlakeIDGeneratorConfig(name string) FlakeIDGeneratorCo
 func makeProxyName(serviceName string, objectName string) string {
 	return fmt.Sprintf("%s%s", serviceName, objectName)
 }
+
+type proxyDestroyer interface {
+	Destroy(ctx context.Context) error
+}
+
+func (m *proxyManager) destroyProxies(ctx context.Context) {
+	for key, p := range m.proxies {
+		if p == nil {
+			m.serviceBundle.Logger.Log(logger.WeightError, func() string {
+				return hzerrors.NewShutdownHandlerError(
```

Comment:
We don't have to create this error, instead we can log it directly. Also we can also ditch the custom error type. Wdyt?

Reply:
- You are right. Since we don't need to identify this error other places. There is no need for creating custom error type.
Fixed, [34fd390](https://github.com/hazelcast/hazelcast-go-client/pull/772/commits/34fd3909b56616404810d1d3e9171fb605f3e865).